### PR TITLE
Disable voting csrf checks

### PR DESCRIPTION
**Pull request by Ian-Foote**
Accessing this url via javascript was failing with a csrf error.

### CODE REVIEW

**Comment by Ian-Foote** [comment]
@incuna/backend Review?

File: bookmarks/urls.py [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
-    (r'^(?P<object_id>\d+)/(?P<direction>up|down|clear)vote/?$',
-        vote_on_object, dict(
-            model=Bookmark,
-            template_object_name='bookmark',
-            template_name='kb/link_confirm_vote.html',
-            allow_xmlhttprequest=True)),
-)
+    url(
+        r'^(?P<object_id>\d+)/(?P<direction>up|down|clear)vote/?$',
+        csrf_exempt(vote_on_object),
```
Comment (KevinEtienne) [inline bookmarks/urls.py]: should vote be in `setup.py`

**Comment by KevinEtienne** [comment]
Looks good to me

File: bookmarks/urls.py (hunk 1 above)
Comment (Ian-Foote) [inline bookmarks/urls.py]: Probably, but that's out-of-scope.

**Pull request merged**